src/modules/item/NavigationButton.tsx

```diff
@@ -0,0 +1,97 @@
+import { useParams } from 'react-router';
+
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward';
+import { AppBar, Box, Toolbar } from '@mui/material';
+
+import { ActionTriggers, DiscriminatedItem, ItemType } from '@graasp/sdk';
+import { Button } from '@graasp/ui';
+
+import { hooks, mutations } from '@/config/queryClient';
+import { useItemContext } from '@/contexts/ItemContext';
+
+const NavigationButton = ({
+  item,
+}: {
+  item: DiscriminatedItem;
+}): JSX.Element | null => {
+  const { rootId } = useParams();
+  const { setFocusedItemId } = useItemContext();
+  const { mutate: triggerAction } = mutations.usePostItemAction();
+  const { data: rootItem } = hooks.useItem(rootId);
+
+  const { data: descendants, isLoading } = hooks.useDescendants({
+    // not correct but enabled
+    id: rootId ?? '',
+    enabled: Boolean(rootId),
+  });
+
+  if (isLoading) {
+    return null;
+  }
```

Comment:
@pyphilia which `if (data)`?

Reply:
The idea is to inverse the order of the checks. It's best to check if the data exists first, rather than checking `isLoading` first. This avoid having unnecessary loader. This logic was not applied everywhere, but we're slowly moving to this strategy. So you're logic should start with `if(descendants)` rather than `if(isLoading)`